manim/mobject/coordinate_systems.py

```diff
@@ -495,3 +503,279 @@ def get_coordinate_labels(self, *numbers, **kwargs):
         self.add(self.get_coordinate_labels(*numbers))
         return self
+
+
+class PolarPlane(NumberPlane):
+    r"""A version of ``NumberPlane`` for polar coordinates.
+
+    Parameters
+    ----------
+    azimuth_line_frequency : :class:`float`, optional
+        The frequency of faded lines in the azimuth, expressed in units of revolution
+        (so :math:`\frac{1}{n}` would result in :math:`n` lines). If ``None`` is specified then it will use the default
+        specified by ``azimuth_units``:
+
+            - ``"PI radians"`` or ``"TAU radians"``: :math:`\frac{1}{20}`
+            - ``"degrees"``: :math:`\frac{1}{36}`
+            - ``"gradians"``: :math:`\frac{1}{40}`
+            - ``None``: :math:`1`
+
+    radius_line_frequency : :class:`float`, optional
+        The frequency of faded lines in the radius.
+
+    azimuth_units : Optional[:class:`str`], optional
+        Specifies a default labelling system for the azimuth. Choices are:
+
+            - ``"PI radians"``: Fractional labels in the interval :math:`\left[0, 2\pi\right]` with :math:`\pi` as a constant.
+            - ``"TAU radians"``: Fractional labels in the interval :math:`\left[0, \tau\right]` (where :math:`\tau = 2\pi`) with :math:`\tau` as a constant.
+            - ``"degrees"``: Decimal labels in the interval :math:`\left[0, 360\right]` with a degree (:math:`^{\circ}`) symbol.
+            - ``"gradians"``: Decimal lables in the interval :math:`\left[0, 400\right]` with a superscript "g" (:math:`^{g}`).
+
+        .. manim:: PolarPlaneUnits
+            :ref_classes: PolarPlane
+
+            polarplot_pi = PolarPlane(azimuth_units="PI radians").scale(0.4).add_coordinates()
+            polarplot_tau = PolarPlane(azimuth_units="TAU radians").scale(0.4).add_coordinates()
+            polarplot_degrees = PolarPlane(azimuth_units="degrees").scale(0.4).add_coordinates()
+            polarplot_gradians = PolarPlane(azimuth_units="gradians").scale(0.4).add_coordinates()
+
+            class PolarPlaneUnits(Scene):
+                def construct(self):
+                    self.add(polarplot_pi)
+                    self.wait(2)
+                    self.play(FadeOutAndShift(polarplot_pi, LEFT))
+                    self.play(FadeInFrom(polarplot_tau, RIGHT))
+                    self.wait(2)
+                    self.play(FadeOutAndShift(polarplot_tau, LEFT))
+                    self.play(FadeInFrom(polarplot_degrees, RIGHT))
+                    self.wait(2)
+                    self.play(FadeOutAndShift(polarplot_degrees, LEFT))
+                    self.play(FadeInFrom(polarplot_gradians, RIGHT))
+                    self.wait(2)
+
+
+    azimuth_offset : :class:`float`, optional
+        The angle offset of the azimuth labels, expressed in radians.
+
+    azimuth_direction : :class:`str`, optional
+        The direction of the azimuth labels.
+
+            - ``"CW"``: Clockwise.
+            - ``"CCW"``: Anti-clockwise.
```

Comment:
In the rendered documentation this list looks like as if it was indented by one level too much; it appears within a blockquote.

```suggestion
        - ``"CW"``: Clockwise.
        - ``"CCW"``: Anti-clockwise.
```